standards/core/peer-exchange.md

```diff
@@ -66,26 +66,45 @@ message PeerInfo {
   bytes enr = 1;
 }
 
-message PeerExchangeQuery {
+message PeerExchangeRequest {
```

Comment:
Not sure if it breaks compatibility by renaming existing fields. Maybe better to keep same name?

Reply:
It should not. I corrected this to follow what is used in nwaku originally. But for protobuf naming does not count but field indices, imho.

Reply:
yea, looks that way...only if you change field numbers then it breaks the existing protocol.